src/function/gds/page_rank.cpp

```diff
@@ -67,9 +75,10 @@ class PageRank final : public GDSAlgorithm {
      * Inputs are
      *
      * graph::ANY
+     * outputProperty::BOOL
```

Comment:
what is `outputProperty`? What property? PageRank property or node properties? Can you provide some documentation?

Reply:
Comment is in gds.h